src/components/slider/slider.component.ts

```diff
@@ -745,6 +746,19 @@ export class SliderComponent implements OnInit, AfterViewInit, DoCheck {
         return destination;
     }
 
+    private updateOrderAtMax(): void {
+        const lowerValue = this.getThumbValue(this.sliderThumb.Lower);
+        const upperValue = this.getThumbValue(this.sliderThumb.Upper);
+        const max = this._options?.track?.max;
+        const maxBuffer = max / 100 * 95;
```

Comment:
This doesn't consider the min value, so you can get stuck at the lower bound as well. Check the last example here:
https://codesandbox.io/s/sliders-ux-aspects-forked-f4ulhh?file=/src/app/app.component.html